src/libraries/Microsoft.Extensions.Logging.EventSource/src/LoggingEventSource.cs

```diff
@@ -298,9 +298,9 @@ internal unsafe void ActivityJsonStop(int ID, int FactoryID, string LoggerName)
         /// <inheritdoc />
         protected override void OnEventCommand(EventCommandEventArgs command)
         {
-            if (command.Command == EventCommand.Update || command.Command == EventCommand.Enable)
+            if (command.Command is EventCommand.Update or EventCommand.Enable)
             {
-                if (!command.Arguments.TryGetValue("FilterSpecs", out string filterSpec))
+                if (!command.Arguments!.TryGetValue("FilterSpecs", out string? filterSpec))
```

Comment:
This is the only place that concerns me. But if it hasn't been a problem, I guess we can just suppress the null check here. Looking around I see similar code elsewhere:

https://github.com/dotnet/runtime/blob/c5d40c9e703fd257db1b26ef4fd1399bbae73ab0/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs#L393-L397

https://github.com/dotnet/runtime/blob/c5d40c9e703fd257db1b26ef4fd1399bbae73ab0/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/CounterGroup.cs#L53-L59